crates/prune/src/segments/headers.rs

```diff
@@ -159,11 +255,17 @@ mod tests {
                 .unwrap_or_default();
 
             let provider = db.factory.provider_rw().unwrap();
-            let result = segment.prune(&provider, input).unwrap();
+            let result = segment.prune(&provider, input.clone()).unwrap();
+            trace!(target: "pruner::test",
+                expected_prune_progress=?expected_result.0,
+                expected_pruned=?expected_result.1,
+                result=?result,
+                "PruneOutput"
+            );
```

Comment:
do we need it in tests?

Reply:
yes, it's needed. when the assertion fails, the fields of the expected `PruneOutput` aren't printed.